replicator/src/test/scala/com/evolutiongaming/kafka/journal/replicator/SKafkaTestUtils.scala

```diff
@@ -0,0 +1,11 @@
+package com.evolutiongaming.kafka.journal.replicator
+
+import com.evolutiongaming.skafka.{Offset, Partition, TopicPartition}
+
+object SKafkaTestUtils {
+  def offset(n: Int): Offset = Offset.unsafe(n)
```

Comment:
why only this one has full nam? shouldn't it be like:
```suggestion
  def o(n: Long): Offset = Offset.unsafe(n)
```

BTW, `Offset` takes `Long`, not `Int`

Reply:
I use only Int's in tests, no point in accepting Long's here.
`o` looks too ambiguous in the test

Reply:
then renaming of `p` to `partition` and `tp` to `topicPartition` should be done to keep consistency

Reply:
no, I like it this way